src/issuers.js

```diff
@@ -0,0 +1,26 @@
+const issuers = [
+  {
+    id: "ca.qc",
+    iss: "XXX",
+    keys: [
+      { kid: "some-kid",
+        alg: "ES256", kty: "EC", crv: "P-256", use: "sig",
+        x: "XSxuwW_VI_s6lAw6LAlL8N7REGzQd_zXeIVDHP_j_Do",
+        y: "88-aI4WAEl4YmUpew40a9vq_w5OcFvsuaKMxJRLRLL0" },
+    ]
+  },
+  {
+    id: "us.ca",
```

Comment:
Can you instead use the iss in the provided QR code and download from there? This will make it work for more than just these 2 states then.

Reply:
I thought about this -- it'd certainly be more convenient for debugging and curiosity-driven inspection.  But anyone can generate codes according to the spec, so you'd end up accepting things generated by, say, fakes.antivaxxers.dumb ... For actual verification purposes, you need a list of trusted issuers even if you fetch the keys on the fly from the spec-mandated location.

Reply:
Yeah, that totally makes sense. Is there anywhere else collecting a list of trusted issuers? This seems like the start of the common root CAs problem with SSL first came along...

Reply:
There's something calling itself the [Common Trust Network](https://www.commontrustnetwork.org/), which unfortunately seems extremely light on details and doesn't seem to have a documented way of fetching the issuers list.  (If you pull apart [their verifier app](https://thecommonsproject.org/smart-health-card-verifier), you can find an undocumented API endpoint to get the list, but (1) I don't know if they actually want other people using it and (2) it appears to cover US issuers only.)

Reply:
There is also [https://github.com/the-commons-project/vci-directory](url) which has this list [https://raw.githubusercontent.com/the-commons-project/vci-directory/main/vci-issuers.json](url)

Reply:
Oh, that's neat, thanks.  (Still seems to cover US issuers only, though.)